examples/cross_build/android/raylib/ci_test_example.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from test.examples_tools import run
+
+# ############# Example ################
+print("- Use the Android NDK to cross-build a package -")
+
+
+profile = """
+
+[settings]
+os=Android
+os.api_level=27
+arch=armv8
+compiler=clang
+compiler.version=18
+compiler.libcxx=c++_static
+compiler.cppstd=17
+build_type=Debug
+
+[conf]
+tools.android:ndk_path={}
+"""
```

Comment:
what about adding this profile as a file in the repo and using jinja to get the environment variable in the profile and advice users to set the variables and modify their profile accordingly?